src/applications/burial-poc-v6/pages/PlotAllowance.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  TextField,
+  Page,
+  RadioGroup,
+} from '@department-of-veterans-affairs/va-forms-system-core';
+import { useFormikContext } from 'formik';
+
+export default function PlotAllowance(props) {
+  const formikContext = useFormikContext();
+
+  return (
+    <div className="vads-u-margin-x--1p5">
+      <Page {...props} title="Plot or interment allowance">
+        <TextField
+          required
+          name="placeOfRemains"
+          label="Place of burial or deceased Veteran’s remains"
+        />
+        <div
+          className={`vads-u-padding-y--1p5 form-expanding-group${formikContext
```

Comment:
Just out of curiosity, what is the difference between the `form-expanding-group` here and the `ExpandingGroupClass` used in the PreviousNames file?

Reply:
Am going to try and refactor to use [ExpandingGroup web-component](https://design.va.gov/storybook/?path=/docs/components-expandinggroup--default)